src/lib.rs

```diff
@@ -61,9 +65,116 @@ impl State {
     }
 }
 
+#[cfg(feature="std")]
+mod oscbuf {
+    use super::MAX_OSC_RAW;
+    pub const DEFAULT_SIZE_LIMIT: usize = u32::max_value() as usize;
+
+    /// When std is available, we can simply use a Vec for the
+    /// OSC storage.  We'll pre-size it to something reasonable.
```

Comment:
It seems like you've put double-spaces behind all full stop in comments. These should all be single spaces following the guideline set by existing comments.

This is true for all comments, I've annotated only this one for convenience.